interfaces/builtin/fwupd.go

```diff
@@ -154,6 +154,10 @@ const fwupdPermanentSlotAppArmor = `
   /sys/devices/**/psp_vbflash rw,
   /sys/devices/**/psp_vbflash_status r,
 
+  # Required by plugin thunderbolt
+  /sys/devices/**/nvm_non_active*/nvmem a,
```

Comment:
do we know if this should maybe allow write instead of only append?

Reply:
I think the only place we use the "non_active" one is here:

https://github.com/fwupd/fwupd/blob/65e5fcf3bef12a475e8b9383bab35775b9139fd2/plugins/thunderbolt/fu-thunderbolt-device.c#L307

Look at the calls to `fu_thunderbolt_device_find_nvmem`. The second parameter is a boolean. If it is "FALSE"  it is a "non_active" one.

Reply:
Ah, now I understand, "a" is only O_APPEND, where was "w" is both. But you cannot have "aw".

Reply:
I still think it would be safer to only allow O_APPEND.